packages/transformer/src/test/standalone/IModelTransformerResumption.test.ts

```diff
@@ -496,6 +496,7 @@ describe.skip("test resuming transformations", () => {
   });
 
   // FIXME: not (yet?) implemented for federation guid optimization branch
+  // FIXME<NICK>: Can I remove both the fixme and possibly even this test file?
```

Comment:
really for 1.0 you can remove the tests and all the resumption APIs

Reply:
I took a pass at that in the commit ''delete resumption.test.ts and resumption APIs"